frontendActions/src/main/java/com/virtuslab/gitmachete/frontend/actions/base/BaseResetBranchToRemoteAction.java

```diff
@@ -124,23 +126,34 @@ public void actionPerformed(AnActionEvent anActionEvent) {
       return;
     }
 
+    var localBranch = getGitMacheteBranchByNameWithLogging(anActionEvent, branchName).getOrNull();
+    if (localBranch == null) {
+      VcsNotifier.getInstance(project).notifyError(VCS_NOTIFIER_TITLE,
+          "Internal error occurred. For more information see IDE log file");
```

Comment:
Yeah, this kind of messages introduced by me was not the best from my part :disappointed: 
Let's reword this to something more useful like `Can't get a local branch '${branchName}'` or something similar.